src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const transponsedRobot = {};
+
+  for (const [key, value] of Object.entries(robot)) {
+    transponsedRobot[value] = key;
+  }
+
+  if (Object.keys(robot).length !== Object.keys(transponsedRobot).length) {
+    return null;
+  }
```

Comment:
Instead of checking for duplicate values after the loop, you can check for duplicates inside the loop. If a duplicate is found, you can return null immediately, making the function more efficient